README.md

```diff
@@ -4,7 +4,7 @@
 
 This is a generic skeleton project that can be used to quickly get a
 new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
+[Packer](https://packer.io) template project started. This skeleton project
```

Comment:
This is not a skeleton repo - the verbiage should reflect that.

Reply:
Yes, I just realized that the `README.md` for this project was never updated from the skeleton boilerplate.  This is taken care of in #57, which can be merged after this PR is merged.